Description: Disable doctests and examples
 The doctests and examples disabled by this patch are actual examples that
 depend on crates not packaged in the archive
Author: NoisyCoil
Forwarded: not-needed
Last-Update: 2024-12-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,6 +21,7 @@
 categories = ["authentication"]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/1wilkens/pam/"
+autoexamples = false
 [dependencies.libc]
 version = "^0.2"
 
@@ -30,7 +31,5 @@
 [dependencies.users]
 version = "0.12"
 package = "uzers"
-[dev-dependencies.rpassword]
-version = "2.0"
 [badges.travis-ci]
 repository = "1wilkens/pam"
--- a/src/authenticator.rs
+++ b/src/authenticator.rs
@@ -14,7 +14,7 @@
 /// want a simple password-based authentication, you can use `Authenticator::with_password`,
 /// and to the following flow:
 ///
-/// ```no_run
+/// ```compile_fail
 /// use pam_auth::Authenticator;
 ///
 /// let mut authenticator = Authenticator::with_password("system-auth")
